service/csv/csv.go

```diff
@@ -0,0 +1,190 @@
+package csv
+
+import (
+	"encoding/csv"
+	"errors"
+	"log"
+	"os"
+	"strconv"
+	"sync"
+
+	"pokemons/model"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Service for Pokemon requests
+type Service struct {
+	logger *logrus.Logger
+	csvr   *os.File
+	csvw   *csv.Writer
+}
+
+// Task - Represents the task for every worker
+type Task struct {
+	ID   int
+	Data model.PokemonsData
+}
+
+// Worker - Worker struct
+type Worker struct {
+	task Task
+}
+
+var (
+	pokemons []*model.PokemonsData
+	rejected = 0
```

Comment:
The values of these variables will persist throughout the execution of your app. Is it ok to keep them here? Wouldn't it make more sense to keep them within the scope of your concurrent function?

Reply:
I though it, but in all the examples I saw, use the global variables and I think is cleaner. 
I was planning to move the concurrency logic to another file, however, the concurrency uses the csv too, so I think move it would be overkill. Another option, if I move the variables to the concurrency functions, I would need to increase the functions' signature, and pass them by reference. 
I would prefer to use the global variables, what do you think?